app/Repositories/AlertGroupsRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace App\Repositories;
+
+use Log;
+use App\Repositories\Eloquent\BaseRepository;
+use App\Repositories\Contracts\RepositoryInterface;
+
+class AlertGroupsRepository extends BaseRepository
+{
+    /**
+     * Specify Model class name
+     *
+     * @return mixed
+     */
+    public function setModel()
+    {
+        return 'App\Models\AlertGroup';
```

Comment:
@TranBinhLuan Change `return 'App\Models\AlertGroup';` to `return App\Models\AlertGroup::class;`